src/js/components/account/login.js

```diff
@@ -17,11 +17,11 @@ export class Login {
     const endPoint = "/api/accounts/login/";
     const fields = [
       {
-        label: "login",
+        label: "Username",
         type: "text",
```

Comment:
I think you need a `name` field here with value `login`

It's trying to pass the field name `Username` to the server, and the server shouldn't care what the field label is in the UI.

Reply:
Doh! You are right.